drivers/hmis/app/graphql/types/hmis_schema/has_referral_requests.rb

```diff
@@ -21,10 +21,11 @@ def referral_requests_field(name, description = nil, type: Types::HmisSchema::Re
         end
       end
 
-      def resolve_referral_requests_with_loader(association_name, **args)
-        scope = scoped_referral_requests(HmisExternalApis::AcHmis::ReferralRequest.all, **args)
-        load_ar_association(object, association_name, scope: scope)
-      end
+      # FIXME: combining the extra scope isn't working here- perhaps rails 6 bug
```

Comment:
Hmm, odd. If it's true what Zach said [that data loaders don't work with pagination](https://github.com/greenriver/hmis-warehouse/pull/3045#discussion_r1191552897), then we wouldn't want to use it here anyway because of that. I added a ticket to the backlog to address our data loaders issues ( / lack of data loader usage) [here](https://www.pivotaltracker.com/story/show/185200747) - if you have any thoughts on that, feel free to comment on the ticket!